### ISSUE TITLE
Parameter wallet_num is not used

### ISSUE THREAD

**Issue by ghtdak** [sbuss/bitmerchant#58]
Not sure if this matters

https://github.com/sbuss/bitmerchant/blob/master/tests/generate_bip32_test_vectors.py#L23

**Comment by sbuss** [comment]
Hi @ghtdak, thanks for the issue! This is definitely a mistake and makes the code hard to follow. I should have removed `wallet_num` from `get_new_address` in c2542110e58dcef61e4f1bf011109e9a7548bc18 since the random seed parameter takes care of it, but I overlooked it.

I'm preparing a fix now.
